### ISSUE TITLE
errors

### ISSUE THREAD

**Issue by GoogleCodeExporter** [manojjsm/js-test-driver#35]
```
Hi!

There's several errors I've found in a day of fiddling with the jstd.

The first was in the eclipse plugin, which died in a nullpointerexception
if a path specified in the config file is not valid. (This is an error on
console too, it shouldn't care if it can't include anything from the
specified path, or it's non-existent)

The second and bigger problem is the assertion of complex structures
(arrays, objects)

expected
{"elements":[{"id":"","innerHTML":"Aladar"},{"id":"","innerHTML":"Bela"},{"id":"
","innerHTML":"Cecil"}]}
but was
{"elements":[{"id":"","innerHTML":"Aladar"},{"id":"","innerHTML":"Bela"},{"id":"
","innerHTML":"Cecil"}]}

All this @HEAD.

Best wishes:
Meza ( @vsbmeza )


```

Original issue reported on code.google.com by `lord.m...@gmail.com` on 30 Aug 2009 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
The second problem (assertion of complex structures) should be solved by a 
patch I
appended to issue: #36 
(http://code.google.com/p/js-test-driver/issues/detail?id=36)

HTH
Jakob
```

Original comment by `jakob.we...@gmail.com` on 2 Sep 2009 at 1:23

**Comment by GoogleCodeExporter** [comment]
```
I've applied Jakob's patch. Do you still have the problem on eclipse with the 
latest
version of the plugin?
```

Original comment by `toc...@gmail.com` on 2 Nov 2009 at 7:02

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in 1.3.0.

Please reopen if necessary.
```

Original comment by `corbinrs...@gmail.com` on 14 Feb 2011 at 4:10
- Changed state: **Fixed**
